spec/webauthn/attestation_statement/android_safetynet_spec.rb

```diff
@@ -49,34 +74,29 @@
     end
 
     before do
-      @_original_trust_store = AndroidSafetynet::AttestationResponse.trust_store
-      AndroidSafetynet::AttestationResponse.trust_store = nil
-    end
-
-    after do
-      AndroidSafetynet::AttestationResponse.trust_store = @_original_trust_store
+      allow(statement).to receive(:attestation_root_certificates).and_return([root_certificate])
     end
 
     let(:credential_key) { OpenSSL::PKey::RSA.new(2048) }
     let(:client_data_hash) { OpenSSL::Digest::SHA256.digest({}.to_json) }
 
     it "returns true when everything's in place" do
-      expect(statement.valid?(authenticator_data, client_data_hash)).to be_truthy
+      expect(statement.valid?(authenticator_data, client_data_hash, timestamp)).to be_truthy
```

Comment:
I get occasional failures for this specific test case...

I consistently get at least 1 failure every 10 runs.

```
$ be rspec spec/webauthn/attestation_statement/android_safetynet_spec.rb

Randomized with seed 34765
F..

Failures:

  1) WebAuthn::AttestationStatement::AndroidSafetynet#valid? returns true when everything's in place
     Failure/Error: expect(statement.valid?(authenticator_data, client_data_hash, timestamp)).to be_truthy
     
       expected: truthy value
            got: false
     # ./spec/webauthn/attestation_statement/android_safetynet_spec.rb:84:in `block (3 levels) in <top (required)>'

Finished in 0.43412 seconds (files took 0.11486 seconds to load)
3 examples, 1 failure

Failed examples:

rspec ./spec/webauthn/attestation_statement/android_safetynet_spec.rb:83 # WebAuthn::AttestationStatement::AndroidSafetynet#valid? returns true when everything's in place

Randomized with seed 34765
```

Reply:
Some weird timing flakiness maybe?

Reply:
Weird, I'll look into it. It shouldn't be timing anymore 😉 Is it consistent with a certain seed or does it somewhat reliably pop up every X runs?

Reply:
Not consistent with fixed seeds.

Reply:
Let me know if you can also reproduce or not..

Reply:
Remembered I ran into this during extraction as well and [used a more specific timestamp](https://github.com/bdewater/safety_net_attestation/blob/d3f859092049c8b56b0e11eb1fd0dfddaeddf57a/spec/statement_spec.rb#L111) in the gem tests to fix - will apply here.